xml/ha_storage_protection.xml

```diff
@@ -781,18 +780,42 @@ Timeout (msgwait)  : 10
        would be appropriate if the <literal>msgwait</literal> timeout value for
        SBD was set to <literal>30</literal> seconds.</para>
      </callout>
-     <callout arearefs="co.ha.sbd.st.rsc">
-      <para>
-       The &stonith; resource to use for SBD.</para>
-     </callout>
-     <callout arearefs="co.ha.sbd.pm_delay_max">
+   </calloutlist>
+  </step>
+  <step>
+   <para>For a two-node cluster, decide if you want predictable or random delays. For
+   other cluster setups you do not need to set this parameter.</para>
+   <variablelist>
+    <varlistentry>
+     <term>Predictable Static Delays</term>
+     <listitem>
+      <para>This parameter enables a static delay before executing &stonith; actions.
+      It ensures that the nodes do not fence each other if separate fencing resources
+      and different delay values are being used. The node with the shorter delay will
+      loose in a <quote>fencing race</quote>.
+      The parameter can be used to <quote>mark</quote> a specific node to survive
+       in case of a split-brain scenario in a two-node cluster.
+      <remark>toms 2018-05-09: From FATE#324488, hmn, not sure how to do that:
+      "They need to configure two stonith resources respectively targeting the two nodes,
+      and adjust "pcmk_delay_base" parameters whenever a SAP master is promoted."</remark>
+      </para>
+      <screen>&prompt.crm.conf;<command>primitive</command> stonith_sbd stonith:external/sbd
+  params pcmk_delay_base=30</screen>
```

Comment:
It's important to clearly mention that  for this case of  static delays,  we should configure separate stonith resources targeting respective nodes, so that different fencing delays can be achieved. And I think the test case from the fate entry also can be a good example to demonstrate it in this doc:
```
primitive st-sbd-alice stonith:external/sbd params pcmk_host_list=alice pcmk_delay_base= 20
primitive st-sbd-bob stonith:external/sbd params pcmk_host_list=bob pcmk_delay_base=0
```

Reply:
Thanks Yan! I've removed the remark and added the two lines with the primitive. Furthermore, I've added a sentence explaining it